tests/BinaryOperationTest.php

```diff
@@ -640,6 +640,14 @@ function foo(int $s1): string {
                         return "foo" . $s1;
                     }',
             ],
+            'numericWithInt' => [
+                '<?php
+                    /** @param numeric $a */
+                    function b($a): int|float{
+                        $a++;
```

Comment:
Perhaps it makes sense to also check a couple more operations, like `-` or `*`

Reply:
done